docs/activities.md

```diff
@@ -39,6 +39,22 @@ Activity Definitions are named and referenced in code by their <a class="tdlp" h
 
 <div class="tdiw"><div class="tditw"><p class="tdit">Activity Definition</p></div><div class="tdiiw"><img class="tdi" src="/diagrams/activity-definition.svg" alt="Activity Definition" /></div></div>
 
+#### Idempotency
+
+Temporal recommends that Activities are idempotent.
+
+Activities are idempotent if multiple applications of that operation do not change the state of the system beyond the initial application.
+
+Activities may be retried repeatedly, so you may need to use idempotency keys for critical side effects.
```

Comment:
To clarify, they may also be executed more than once even if the retry policy says they shouldn't. So it's not just "retry".